tests/conftest.py

```diff
@@ -89,8 +90,10 @@ def _test_env(monkeypatch):
     monkeypatch.setenv("SENTRY_DSN", "None")
     monkeypatch.setenv("WORKSPACE", "test")
     monkeypatch.setenv("AWS_REGION_NAME", "us-east-1")
+    monkeypatch.setenv("AWS_DEFAULT_REGION", "us-east-1")
```

Comment:
I think some AWS services use `AWS_REGION_NAME` and others use `AWS_DEFAULT_REGION` to set default region values. When running DSC in AWS, it will use the appropriate env var, which is why I didn't feel it was necessary to add `AWS_DEFAULT_REGION` to `dsc.config.Config`. 🤔